packages/synapse-react-client/src/components/SynapseTable/SynapseTable.tsx

```diff
@@ -128,7 +127,7 @@ export function SynapseTable(props: SynapseTableProps) {
       )
     : undefined
 
-  const columns: ColumnDef<Row, string | null>[] = useMemo(
+  const columns = useMemo(
```

Comment:
This type assignment started causing problems, so just remove it and stick with the inferred type